src/lib/rehype-sectionise.ts

```diff
@@ -0,0 +1,52 @@
+import type { Element, ElementContent, Root } from "hast";
```

Comment:
Nice work with this plugin!
Please add something to the README explaining the use of this new plugin for ReactMarkdown